adjusted the error message of "more than 5 queries needed for exchange"

hippocritical opened the pull request:
## Summary
I found myself second guessing what the exchanges candle amount per query actually is and seeing those error messages.
Without debugging there is no way for the user to actually know what it is.
Now users don't have to guess what the limit actually is to then work towards, helping to better understand the error message.

hippocritical committed:
adjusted the error message of "more than 5 queries needed for exchange" to be more verbose to the user.
Now they don't have to guess what the limit actually is to then work towards, helping to better understand the error message.

hippocritical committed:
didn't make a space in the end ... you can book that under typo ...

added a . at the end of another warning since all other messages do have the same structure.

xmatthias commented on freqtrade/exchange/exchange.py:
```diff
@@ -880,7 +880,8 @@ def validate_required_startup_candles(self, startup_candles: int, timeframe: str
                     "which is more than 5x "
-                    f"the amount of candles {self.name} provides for {timeframe}."
+                    f"the amount of candles {self.name} provides for {timeframe} "
+                    f"at a startup_candle_count limit of {candle_limit * 5 - 1}."
```
what about the following (can't do suggestion as it'd change things that were not changed 😆 )

```
f"This strategy requires {startup_candles} candles to start, "
f"which is more than 5x ({candle_limit * 5 - 1} candles) "
f"the amount of candles {self.name} provides for {timeframe}."
```

it keeps the sentence short - and i find the current version reads kinda odd somehow - 
"at startup_candle_count limit of 1000." is not pretty clear if you ask me.

hippocritical replied:
sure, I just tried to append it, doing it your way looks better.

        elif required_candle_call_count > 1:
            raise ConfigurationError(
                f"This strategy requires {startup_candles} candles to start, which is more than "
                f"the amount of candles {self.name} provides for {timeframe}."
            )

not sure about this one though, would the change be here proper, too?
here just with a mentioning of the maximum candles of the exchange -1 instead of 5*

hippocritical committed:
changed the wording in
if required_candle_call_count > 5:
as per matthias suggestion.

adjusted the elif part too, since this would have to be worded similarly.

If a native speaker thinks there is a better wording, be our guest.

hippocritical committed:
fixed tests according to failed tests on github

xmatthias reviewed (approved):
Great! 👍
